### DISCUSSION TITLE
Have a trouble with 'contiguous' option

### DISCUSSION THREAD

**Discussion by microhumanis** [inducer/pymetis#40] [Troubleshooting]
Hi, while I am using the 'part_graph' command to divide graph into contiguous partitions, I faced a error 'TypeError: part_graph() got an unexpected keyword argument 'contiguous'' 

How can I handle this problem?

Thank you :)

**Comment by inducer** [comment]
What version of pymetis? Installed from where?

**Comment by microhumanis** [comment]
Thank you for your reply :)

First, I would like to apologize if I offended you. I didn't know posting questions like this would offend you because I'm not used to Github.

I used PyMetis version 2020.1 and installed pymetis by using '$ pip install pymetis' command.

I found that basic 'part_graph' command (1) works well, but command with contiguous option raises an error 'TypeError: part_graph() got an unexpected keyword argument 'contiguous'.


(1) n_cuts, membership = pymetis.part_graph(2, adjacency=adjacency_list)

(2) n_cuts, membership = pymetis.part_graph(2, adjacency=adjacency_list, contiguous=True)

Thank you very much for your time. I look forward to hearing from you.

Best Regards,

Microhumanis

2022. 7. 20. 오후 10:01, Andreas Klöckner ***@***.******@***.***>> 작성:



What version of pymetis? Installed from where?

**Comment by inducer** [comment]
The version on the package index did not yet have the `contiguous` argument. I've just released v2022.1, which does include it:

https://pypi.org/project/PyMetis/2022.1/

**Comment by microhumanis** [comment]
Thank you for all your assistance. I really appreciate your help.

Best Regards,

Microhumanis

2022. 7. 24. 오전 3:46, Andreas Klöckner ***@***.******@***.***>> 작성:



The version on the package index did not yet have the contiguous argument. I've just released v2022.1, which does include it:

https://pypi.org/project/PyMetis/2022.1/
